lib/order.rb

```diff
@@ -0,0 +1,38 @@
+require_relative './dish.rb'
+require 'rubygems'
+require 'twilio-ruby'
+
+# put your own credentials here
+account_sid = 'AC8987a086c237f27afe114b616e04d0eb'
+auth_token = 'd7ccfed6ac5e24e0d862d78a056b1092'
+
+# set up a client to talk to the Twilio REST API
+@client = Twilio::REST::Client.new account_sid, auth_token
+
+class Order
+  attr_reader :dishes_ordered
+
+  def initialize(*dishes)
+    @dishes_ordered = dishes
+    @order_total = total
```

Comment:
I like that this isn't dependent on instances from another class. I don't know what works better in terms of dependency but this seems simpler